<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QDialog" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pushNotifications">
      <attribute name="title">
       <string>Push notifications</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="pushNotificationLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>General</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="pcr">
              <property name="text">
               <string>Pending contact request</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="shares">
              <property name="text">
               <string>Shares</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutc">
              <item>
               <widget class="QLabel" name="labelc">
                <property name="text">
                 <string>Global chat notifications enabled</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="globalChatNotificationsEnabled">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacerc">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelc1">
                <property name="text">
                 <string>Do not disturb period(s):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="globalChatDnd"/>
              </item>
              <item>
               <spacer name="horizontalSpacerc1">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Global notifications enabled        </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="globalNotificationsEnabled">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Do not disturb period(s):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="globalDnd"/>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Schedule Start:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="scheduleEnabled">
                <property name="text">
                 <string>Enabled</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="startTime">
                <property name="maximumSize">
                 <size>
                  <width>125</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>End:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="endTime">
                <property name="maximumSize">
                 <size>
                  <width>125</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Time Zone:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="timeZones">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Chats</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QListView" name="chatsView"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="presence">
      <attribute name="title">
       <string>Presence</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>951</width>
         <height>491</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="rowWrapPolicy">
         <enum>QFormLayout::DontWrapRows</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="statusComboBox">
          <property name="minimumSize">
           <size>
            <width>789</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Offline</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Away</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Online</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Busy</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="autoAwayLabel">
          <property name="text">
           <string>Auto-away:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="autoAwayCheckBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="autoAwayTimeoutSLabel">
          <property name="text">
           <string>Auto-away timeout (s):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="autoAwayTimeoutSLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="showLastGreenLabel">
          <property name="text">
           <string>Show last green:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="showLastGreenCheckBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="persistStatusLabel">
          <property name="text">
           <string>Persist status:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="persistStatusCheckBox"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="confirmButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButtons</sender>
   <signal>accepted()</signal>
   <receiver>SettingWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButtons</sender>
   <signal>rejected()</signal>
   <receiver>SettingWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
